assets/app/models/Github.js

```diff
@@ -116,9 +129,9 @@ var GithubModel = Backbone.Model.extend({
           url: url,
           complete: function(res) {
             var r = {
-              file: file,
+              file: file.name,
               present: (res.status === 200) ? true : false,
-              json: res.responseJSON || []
+              json: res.responseJSON || yaml.parse(res.responseText)
```

Comment:
This throws an error when one of the config / navigation files is not available:

```
GET http://localhost:1337/site/dhcole/microsite-template/_navigation.json 404 (Not Found)send @ bundle.js:24812jQuery.extend.ajax @ bundle.js:24348(anonymous function) @ bundle.js:246(anonymous function) @ bundle.js:2335(anonymous function) @ bundle.js:1885_arrayEach @ bundle.js:1737_each @ bundle.js:1728async.forEachOf.async.eachOf @ bundle.js:1884_parallel @ bundle.js:2334async.parallel @ bundle.js:2348Backbone.Model.extend.fetchConfig @ bundle.js:260Backbone.Model.extend.initialize @ bundle.js:145Backbone.Model @ bundle.js:3259child @ bundle.js:4703Backbone.View.extend.initialize @ bundle.js:823Backbone.View @ bundle.js:4043child @ bundle.js:4703Backbone.View.extend.edit @ bundle.js:1194Backbone.Router.extend.edit @ bundle.js:47_.extend.execute @ bundle.js:4336(anonymous function) @ bundle.js:4324(anonymous function) @ bundle.js:4607_.some._.any @ bundle.js:45924_.extend.loadUrl @ bundle.js:4605(anonymous function) @ bundle.js:28(anonymous function) @ bundle.js:3183(anonymous function) @ bundle.js:46556triggerEvents @ bundle.js:3225triggerApi @ bundle.js:3211eventsApi @ bundle.js:3009Events.trigger @ bundle.js:3201_.extend.set @ bundle.js:3381options.success @ bundle.js:3448fire @ bundle.js:19281self.fireWith @ bundle.js:19393done @ bundle.js:24446(anonymous function) @ bundle.js:24787
bundle.js:47951 Uncaught <ParseException> You cannot define a sequence item when in a mapping
``